test/e2e_tests/dir_test.go

```diff
@@ -120,22 +123,26 @@ func (suite *dirTestSuite) TestDirCreateDuplicate() {
 
 // # Create Directory with special characters in name
 func (suite *dirTestSuite) TestDirCreateSplChar() {
-	dirName := suite.testPath + "/" + "@#$^&*()_+=-{}[]|?><.,~"
-	err := os.Mkdir(dirName, 0777)
-	suite.Equal(nil, err)
+	if !suite.fileShareTest {
```

Comment:
Is this documented somewhere like what all are valid characaters in file name ? If so, we shall mention the link somewhere here in comments saying this is the reason we are disabling this test case for file-share.

Reply:
Yes, it's documented in the release-distro-notes.yml (lines 167-169) inside the azure-pipeline-templates directory. I included the documentation link in the comments.

Reply:
I would document it in this test file over the yml file.